github/release_notes/model.py

```diff
@@ -71,7 +71,9 @@ class ReleaseNote:
     user_login: str
     is_current_repo: bool
```

Comment:
I have some questions w.r.t. this dataclass.

- what does `is_current_repo` mean? also: since there are now source_component_name and current_component_name, can this attr not be calculated (if it is at all required)?
- what is user_login used for?